src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -294,7 +294,7 @@ public GameObject InstantiateObject(string typeName)
             var objectDefinition = IniDataContext.Objects.FirstOrDefault(x => x.Name == typeName);
             if (objectDefinition != null)
             {
-                return new GameObject(objectDefinition, this, _game.CivilianPlayer);
+                return new GameObject(objectDefinition, this, _game.CivilianPlayer, null);
```

Comment:
Because I didn't feel like passing the GameObjectCollection through the whole call struct, and it doesn't seem to be necessary for civilian objects yet

Reply:
Fixed (those function calls cascaded to other function calls, but handled those as well)